packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue

```diff
@@ -360,6 +361,7 @@ const selectInputProps = computed(() => {
                 :active="option.value === activeValue"
                 :icon="option.icon"
                 :density="props.density"
+                :truncation="props.truncation"
```

Comment:
could you please add a storybook example? did you test it in combination with the custom-options?

Reply:
info: it works properly with custom-options 👍
![image](https://github.com/SchwarzIT/onyx/assets/151019977/26b6b491-1eae-4424-a0dd-eb156ac5a14a)

but having one storybook example with multiline flyout would be helpful :)